README.md

```diff
@@ -180,7 +181,7 @@ Integration written by [Miepee](https://github.com/Miepee). Offworld sprites are
 ## Setup
 
 Getting started:
-   1. Clone this repository (downloading the zip is *not* supported and will not work)
+   1. Clone this repository. You will also need to pull the tags after, which you can do via `git pull --tags`. (downloading the zip is *not* supported and will not work)
```

Comment:
Tags come by default when cloning. The usual issue is when people fork without tags, which is default I believe.

Reply:
Should I remove this addition, then?

Reply:
If tags are going to be mentioned, I'd prefer to have an explicit guide for forking the repository

Reply:
OK, I'll take it out then. Someone who knows what they're talking about (not me) should be the one to write the forking guide :p